docs/DEV_ENV.md

```diff
@@ -1,72 +1,82 @@
 # Local Development Environment
 
-This uses the [Happy Path setup](https://wiki.czi.team/display/TECH/Data+Portal+Happy+Path+--+How+it+works) created by the shared infra team. 
+This uses the [Happy Path setup](https://wiki.czi.team/display/TECH/Data+Portal+Happy+Path+--+How+it+works) created by the shared infra team.
 
 ## Development quickstart
 
 1. [install docker](https://docs.docker.com/get-docker/). If brew is installed run `brew install docker`.
+1. [install pre-commit](https://pre-commit.com/#install). `pip install pre-commit` or `brew install pre-commit`
```

Comment:
Add step to install `pre-commit` (I didn't do this, which was why it wasn't working for me 😆 )